sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1618,14 +1751,21 @@ def task_worker(self):
                                    self.log_error("{} failed to configure laser frequency {} GHz".format(lport, freq))
                                 else:
                                    self.log_notice("{} configured laser frequency {} GHz".format(lport, freq))
-
+                        
+                        helper_logger.log_debug("Starting to stage custom SI settings")
                         # Stage custom SI settings
                         if optics_si_parser.optics_si_present():
                             optics_si_dict = {}
                             # Apply module SI settings if applicable
                             lane_speed = int(speed/1000)//host_lane_count
                             optics_si_dict = optics_si_parser.fetch_optics_si_setting(pport, lane_speed, sfp)
-
+                            
+                            helper_logger.log_debug("SI values for the connected module found in optics_si_settings.json:")
```

Comment:
are you going to keep this debug code?

Reply:
I updated the log message according to Moshe's suggestion